### PR TITLE
Release notes and housekeeping.

### PR DESCRIPTION
**Pull request by arthurevans**
Many deferred release notes. Updates to settings and gestures docs. Also minimal front-page and community page updates.

### CODE REVIEW

File: app/2.0/docs/devguide/settings.md [hunk 1]
```diff
@@ -4,9 +4,45 @@ title: Global Polymer settings
 
 Document-level global Polymer settings can be set
 by creating a `Polymer` object on window before importing the Polymer
-library:
+library, or by calling a setter.
 
+
```
Comment (arthurevans) [inline app/2.0/docs/devguide/settings.md]: @azakus can you review this file? Trying to add new instructions/samples for calling the setter.

File: app/2.0/docs/devguide/gesture-events.md [hunk 2]
```diff
@@ -188,12 +188,52 @@ to be garbage collected.
 
 ```
 
-## Gestures and scroll direction
+## Native browser gesture handling {#gestures-and-scroll-direction}
+
```
Comment (arthurevans) [inline app/2.0/docs/devguide/gesture-events.md]: @azakus this section is new, replacing some stale 1.x stuff.

**Comment by arthurevans** [comment]
Staged at https://2-2-relnotes-dot-polymer-project.appspot.com

File: app/2.0/docs/devguide/gesture-events.md [hunk 3]
```diff
@@ -188,12 +188,52 @@ to be garbage collected.
...
+Browsers implement native handling for certain gestures, such as touch-based scrolling, or letting the user zoom content with a pinch gesture.
+
+Listening for gesture events disables native browser gesture handling by default. For example, nodes with a listener for the `track` event prevent the browser from handling scrolling and pinch-zoom gestures. 
+
+If you want get Polymer gesture events _and_ native gesture handling, you can use the `Polymer.Gestures.setTouchAction` function to specify which events the browser should handle natively. For example, if you want the browser to handle vertical scrolling, but have your element handle left-right swiping actions, you could do something like this:
```
Comment (katejeffreys) [inline app/2.0/docs/devguide/gesture-events.md]: >If you want get Polymer gesture events _and_ native gesture handling, 

Suggest rewording, maybe

>If you want to use Polymer gesture events _and_ native gesture handling,

File: app/blog.yaml [hunk 4]
```diff
@@ -1,4 +1,9 @@
 ---
+- title:  "Upcoming changes to shadow DOM v0 and HTML imports"
```
Comment (katejeffreys) [inline app/blog.yaml]: This gets awkwardly truncated from 

>"Upcoming changes to shadow DOM v0 and HTML imports"

to 

>"Upcoming changes to shadow DOM v0 and HTML imp..."

An option for a shorter title could be:

>"Shadow DOM v0 and HTML imports are changing"

File: app/blog/2017-10-18-upcoming-changes.md [hunk 5]
```diff
@@ -0,0 +1,146 @@
+---
+title:  "Changes to shadow DOM v0 and HTML imports"
+---
+
+
+In the near future, Chrome is rolling out two changes that may affect existing Polymer 1.x and 2.x websites:
+
+
+*   Shadow-piercing selectors (`/deep/` and `::shadow`). Chrome 63 will disable the shadow-piercing selectors in style sheets. This change only affects sites using Polymer 1.x that are using native shadow DOM.
```
Comment (katejeffreys) [inline app/blog/2017-10-18-upcoming-changes.md]: Suggest

>Shadow-piercing selectors (`/deep/` and `::shadow`) will be disabled.

Instead of 

>Shadow-piercing selectors (`/deep/` and `::shadow`).

File: app/blog/2017-10-18-upcoming-changes.md [hunk 6]
```diff
@@ -0,0 +1,146 @@
...
+
+*   HTML imports styling the main document. Chrome 65 will make a change to the behavior of HTML imports. Currently, styles from an HTML import are automatically applied to the main document. This behavior is going away in Chrome 65. This change affects sites using either Polymer 1.x or Polymer 2.x.
```
Comment (katejeffreys) [inline app/blog/2017-10-18-upcoming-changes.md]: Suggest 

>HTML imports will no longer automatically style the main document.

Instead of

>HTML imports styling the main document.

File: app/blog/2017-10-18-upcoming-changes.md [hunk 7]
```diff
@@ -0,0 +1,146 @@
...
+*   Remove the use of `/deep/` and `::shadow` (recommended).
+
+*   Use shady DOM instead of native shadow DOM.
+
+
+### Removing /deep/ and ::shadow
+
+The best fix is to eliminate any remaining use of `/deep/` and `::shadow` in your sites. The `::shadow` pseudo class has been fairly lightly used, but `/deep/` was used in the past for theming.
+
+For example, there was a previous version of the iron-flex-layout import that used `/deep/`. Search for the following string:
```
Comment (katejeffreys) [inline app/blog/2017-10-18-upcoming-changes.md]: Suggest using backticks around iron-flex-layout

>For example, there was a previous version of the `iron-flex-layout` import that 

Instead of 

>For example, there was a previous version of the iron-flex-layout import that

File: app/blog/2017-10-18-upcoming-changes.md [hunk 8]
```diff
@@ -0,0 +1,146 @@
...
+  <script>
+    window.Polymer = {
+      dom: 'shadow',
+      lazyRegister: true
+    };
+  </script>
+```
+
+
+The settings script is usually in your entrypoint, since it needs to run before you load Polymer or any polymer element definitions.
```
Comment (katejeffreys) [inline app/blog/2017-10-18-upcoming-changes.md]: suggest capitalizing Polymer

>before you load Polymer or any Polymer element

instead of 

>before you load Polymer or any polymer element

File: app/blog/2017-10-18-upcoming-changes.md [hunk 9]
```diff
@@ -0,0 +1,146 @@
...
+
+
+```
+[Deprecation] Styling master document from stylesheets defined in HTML Imports is deprecated, and is planned to be removed in M65, around March 2018. Please refer to https://goo.gl/EGXzpw for possible migration paths.
+```
+
+
+The good news is we already have a fix for this in the latest versions of Polymer 1.x and 2.x. The bad news is that we can't make the warning go away. (We're looking into a workaround to suppress this warning, which could be shipped in a subsequent version of Polymer.)
+
+With the fix, the custom-style element moves its styles into the main document. Unfortunately, the custom style element can't do that until after the browser has parsed the HTML import and printed the warning. So you'll continue to see the warning after upgrading.
```
Comment (katejeffreys) [inline app/blog/2017-10-18-upcoming-changes.md]: >With the fix, the `custom-style` element

Instead of 

>With the fix, the custom-style element

File: app/blog/2017-10-18-upcoming-changes.md [hunk 10]
```diff
@@ -0,0 +1,146 @@
...
+
+What you should do in the next month or two:
+
+
+
+*   Ensure that you're running the latest 1.x or 2.x release (1.10.1 or newer for 1.x; 2.1.1 or newer for 2.x).
+
+*   If you have any HTML imports that contain ordinary style tags—not wrapped in a `custom-style` element or inside a template—use a custom-style element instead, to ensure the styles are moved into the main document.
```
Comment (katejeffreys) [inline app/blog/2017-10-18-upcoming-changes.md]: >use a `custom-style` element instead

instead of 

>use a custom-style element instead

File: app/2.0/docs/devguide/gesture-events.md (hunk 2 above)
Comment (azakus) [reply app/2.0/docs/devguide/gesture-events.md]: LGTM

File: app/2.0/docs/devguide/settings.md (hunk 1 above)
Comment (azakus) [reply app/2.0/docs/devguide/settings.md]: LGTM

**Comment by arthurevans** [comment]
Thanks all!
